jwst/ramp_fitting/tests/test_ramp_fit_step.py

```diff
@@ -246,3 +250,117 @@ def test_int_times2(generate_miri_reffiles, setup_inputs):
     assert cube_model is not None
 
     assert(len(cube_model.int_times) == nints)
+
+
+def one_group_suppressed(suppress, setup_inputs):
+    """
+    Tests three pixel ramps.
```

Comment:
Should that be "Creates three pixel ramps" ?

Reply:
Changed wording.